spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+RSpec.describe UsersController, type: :controller do
+  describe 'GET index' do
+    it 'assigns @users' do
+      user = User.create(name: 'John Doe')
+      get :index
+      expect(response).to be_successful
+      expect(assigns(:users)).to eq([user])
+    end
+  end
+end
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that **If response status was correct**, **If a correct template was rendered** and **If the response body includes correct placeholder text.** So on above test case you check If the response body includes correct placeholder text So Kindly include the test case to check  **If response status was correct**, **If a correct template was rendered** Kindly just like the below sample 
```
 it 'responds with message success' do
      expect(response).to have_http_status(:ok)
    end

    it 'renders users list view' do
      expect(response).to render_template(:index)
    end
```
- [ ] Kindly I recommend you to include test case for **show actions** for User controllers as you write the test case for index actions Just like the below sample and kindly also include the above list three scenario

```
describe 'GET /show' do
  ...
end
 
```